spatial/r3/vector_test.go

```diff
@@ -288,3 +309,32 @@ func vecApproxEqual(a, b Vec, tol float64) bool {
 		scalar.EqualWithinAbs(a.Y, b.Y, tol) &&
 		scalar.EqualWithinAbs(a.Z, b.Z, tol)
 }
+
+var vectorFields = []struct {
```

Comment:
Please put this in front of the test function. Are you planning to use this for other tests? If not please s/vectorFields/divergenceTests/.

Reply:
I'll move them. I am planning on using them for other tests that require vector fields. i.e. jacobian calculation.